### DISCUSSION TITLE
LQI shows 0

### DISCUSSION THREAD

**Discussion by dsduarte** [Koenkk/zigbee2mqtt#9084] [General]
Hello... can anybody help with a thing?
On my devices tab, all my devices shows LQI 0. I assume LQI=0 means that there is no estabelished connection but they are working just fine and on network map, the show connected with LQI different from 0.
![devices](https://user-images.githubusercontent.com/19639490/136601272-33a647b7-e683-4ffc-a9ae-e7614e77f2a1.png)
![map](https://user-images.githubusercontent.com/19639490/136601277-7476036c-3420-4a65-b80a-40ca876dea8e.jpg)

**Comment by Koenkk** [comment]
Are you using a Conbee II as coordinator? If yes this is a known bug: https://github.com/Koenkk/zigbee2mqtt/issues/6776

**Comment by dsduarte** [comment]
Hi... it's the CC2531... is this a conbee II?

Here what show in the about page on Zigbee2Mqtt settings:

Zigbee2MQTT version
1.21.2 commit: unknown
Coordinator type
zStack12
Coordinator revision
20201128
Frontend version
0.6.12

**Comment by dsduarte** [comment]
also, if it helps.... its running on HA addon... HA is runing as a VM with haas.io on a QNAP NAS.

**Comment by Koenkk** [comment]
CC2531 != Conbee, then it's probably a bug in the firmware of the sonoff device, not something that can be fixed from z2m (also not something to worry about when it works fine)

**Comment by dsduarte** [comment]
I removed z2m addon and installed again... a fresh new.. and I think could be some other thing...
I paired two SNZB-02... T&H sensors... after all night after pairing, they are reporting LQI but not reporting temperature, humidity nor battery status:
![image](https://user-images.githubusercontent.com/19639490/136694668-50d0f66b-7734-4d7b-985a-e2b84a801c53.png)
![image](https://user-images.githubusercontent.com/19639490/136694747-e1b9db52-e979-478a-b2e7-6cee3b3943f6.png)

Sorry... I'm very new at this and I can not imagine how to troubleshooting this.

**Reply by Koenkk** [reply]
Working on this in #8971

**Reply by dsduarte** [reply]
Hi... thanks a lot for your good work...
I don't know if can be the same problem but a noticed that not only SNZB-02's have reporting issues... I'll compile a list:
1- SNZB02's are not reporting temperature, humiduty, LQI, and battery info;
2- ZBMIN's and BASICZBR3 are not reporting LQI;
3- SNZB01 (action switch) are not reporting LQI, battery info and action button;
4- SNZB03 (PIR sensor) are reporting battery and occupancy, but not reporting LQI;
5- SNZB04 (contact sensor) are reporting battery and contact status, but not reporting LQI.
6- Tuya Scene Swich (TS0044) is not reporting LQI.
![image](https://user-images.githubusercontent.com/19639490/136831640-17502dcf-44f1-4dbf-86d4-556749cf0847.png)

**Comment by eigenphase** [comment]
I have the same issue. I am on the latest zigbee2mqtt and Sonoff Zigbee Adapter with latest firmware (as recommended by zigbee2mqtt):

<img width="646" alt="image" src="https://user-images.githubusercontent.com/7464149/204106904-38cd87cc-5122-469d-a1a4-155428678d05.png">

Are there any updates on this?
